src/main.rs

```diff
@@ -169,8 +174,101 @@ async fn main() -> Result<(), Box<dyn Error>> {
                 .await?;
         }
         cli::Commands::Add => println!("not yet implemented"),
-        cli::Commands::Edit => println!("not yet implemented"),
+        cli::Commands::Edit { activity } => {
+            let activity = promp_activitie_select(&moco_client, activity).await?;
+
+            let now = Utc::now().format("%Y-%m-%d").to_string();
+
+            print!("New date (YYYY-MM-DD) - Default '{}': ", activity.date);
+            std::io::stdout().flush()?;
+
+            let date = utils::read_line()?;
+            if date.is_empty() {
+                now.clone()
+            } else {
+                date.clone()
+            };
+
+            print!("New duration in hours - Default '{}': ", activity.hours.to_string());
+            std::io::stdout().flush()?;
+
+            let hours = utils::read_line()?;
+            if hours.is_empty() {
```

Comment:
This if else makes no sense because you are not saving the return value of this if else.